File: week_17/D4/practice-for-week-17-python-import-practice-main/python/office/idle.py

# Your code here
import random

def chat():
    coworkers = ["Jack", "Lenny", "Michelle", "Andrea"]
    chatee = coworkers[random.randint(0,3)]
    print("Chatting with " + chatee + "...")
    print("Done")

def getWater():
    print("Getting water...")
    print("That was refreshing.")

def useSocialMedia():
    socialMedia = ["FaceBook", "Twitter", "YouTube", "Reddit"]
    choice = socialMedia[random.randint(0,3)]
    print("Using " + choice + "...")
    print("Done")

File: week_17/D4/practice-for-week-17-python-import-practice-main/javascript/office/idle.js
const chat = () => {
    const coworkers = ["Jack", "Lenny", "Michelle", "Andrea"];
    let chatee = coworkers[Math.floor(Math.random()*4)];
    console.log("Chatting with " + chatee + "...");
    console.log("Done");
}

const getWater = () => {
    console.log("Getting water...");
    console.log("That was refreshing.");
}

const useSocialMedia = () => {
    const socialMedia = ["FaceBook", "Twitter", "YouTube", "Reddit"];
    let choice = socialMedia[Math.floor(Math.random()*4)];
    console.log("Using " + choice + "...");
    console.log("Done");
}

module.exports = {
    chat,
    getWater,
    useSocialMedia
}